hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/BlockDeletingService.java

```diff
@@ -127,12 +128,14 @@ public Long getNumBlocksToDelete() {
   public BackgroundTaskQueue getTasks() {
     BackgroundTaskQueue queue = new BackgroundTaskQueue();
 
+    metrics.resetLastIterationCounts();
```

Comment:
If someone queries the metrics after the `resetLastIterationCounts` is done and before we populate all the counters again, won't we get inconsistent result?

Reply:
I have changed it now, to store the counter and update the metrics only at the end of each iteration. COuld you please take a look at the update?